go/Makefile

```diff
@@ -114,7 +114,7 @@ format:
 unit-test:
 	mkdir -p reports
 	set -o pipefail; \
-	go test -v ./... -skip 'Example|TestGlideTestSuite' $(if $(test-filter), -run $(test-filter)) \
+	go test -v ./... -skip 'Example|TestGlideTestSuite|TestAdvancedTestSuite' $(if $(test-filter), -run $(test-filter)) \
```

Comment:
Please use a test flag. Create a new flag for new test suite. It makes ease test selection.
See how @jbrinkman added a new flag for pubsub

Reply:
See commit https://github.com/valkey-io/valkey-glide/pull/3708/commits/275f33c8e9494a85cf3450e8c1e66e0c00a3eec9